ops/terraform/services/pipeline/README.md

```diff
@@ -63,7 +63,6 @@ https://terraform-docs.io/user-guide/configuration/
 | [aws_launch_template.this](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/launch_template) | resource |
 | [aws_s3_bucket.ccw-verification](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket) | resource |
 | [aws_s3_bucket.this](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket) | resource |
-| [aws_s3_bucket_acl.this](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_acl) | resource |
```

Comment:
I re-ran `terraform-docs` in case this file needed to be updated. Nothing relevant to this PR was changed, but it seems this param was removed a while ago and the doc was never updated.